test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,23 @@
+package codeu.chat.util;
+import java.io.IOexception;
+import static org.junit.Assert.*;
+public final class TokenizerTest {
+	@Test
+	public void testWithNoQuotes() throws IOexception {
```

Comment:
Same thing here, after `throws` it should read `IOException`. The E needs to be capitalized every time.